src/GraphQLParser/Exceptions/GraphQLMaxDepthExceededException.cs

```diff
@@ -0,0 +1,16 @@
+namespace GraphQLParser.Exceptions
+{
+    /// <summary>
+    /// An exception representing a GraphQL document syntax error.
```

Comment:
Copy-pasted comment :(

Reply:
```suggestion
    /// An exception representing a GraphQL 'maximum depth exceeded' error.
```

Reply:
> Copy-pasted comment :(

Yeah I've been working on way too many PRs today!  Getting lost in the changes...not reviewing my own work...etc

Reply:
It may seem that this is a standard error from the spec.
```suggestion
    /// An exception representing a 'maximum depth exceeded' parser error.
```